test/parallel/test-permission-child-process-inherit-flags.js

```diff
@@ -0,0 +1,105 @@
+// Flags: --permission --allow-child-process --allow-fs-read=* --allow-worker
+'use strict';
+
+const common = require('../common');
+const { isMainThread } = require('worker_threads');
+
+if (!isMainThread) {
+  common.skip('This test only works on a main thread');
+}
+
+if (!common.hasCrypto) {
+  common.skip('no crypto');
+}
```

Comment:
Can you add a comment in here about why/how this test depends on crypto, as it's not obvious.